pkg/yurtctl/cmd/clusterinfo/clusterinfo.go

```diff
@@ -0,0 +1,121 @@
+/*
+Copyright 2020 The OpenYurt Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package clusterinfo
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	ct "github.com/daviddengcn/go-colortext"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/klog"
+
+	"github.com/alibaba/openyurt/pkg/projectinfo"
+	kubeutil "github.com/alibaba/openyurt/pkg/yurtctl/util/kubernetes"
+
+)
+
+// ClusterInfoOptions has the information that required by cluster-info operation
+type ClusterInfoOptions struct {
+	clientSet             *kubernetes.Clientset
+	CloudNodes            []string
+	EdgeNodes             []string
+	YurtNodes	          []string
+}
+
+// NewClusterInfoOptions creates a new ClusterInfoOptions
+func NewClusterInfoOptions() *ClusterInfoOptions {
+	return &ClusterInfoOptions{
+		CloudNodes: []string{},
+		EdgeNodes: []string{},
+		YurtNodes: []string{},
+	}
+}
+
+// NewClusterInfoCmd generates a new cluster-info command
+func NewClusterInfoCmd() *cobra.Command {
+	o := NewClusterInfoOptions()
+	cmd := &cobra.Command{
+		Use:   "cluster-info",
+		Short: "list cloud nodes and edge nodes in cluster",
+		Run: func(cmd *cobra.Command, _ []string) {
+			if err := o.Complete(cmd.Flags()); err != nil {
+				klog.Fatalf("fail to complete the cluster-info option: %s", err)
+			}
+			if err := o.Run(); err != nil {
+				klog.Fatalf("fail to run cluster-info cmd: %s", err)
+			}
+		},
+	}
+
+	return cmd
+}
+
+// Complete completes all the required options
+func (o *ClusterInfoOptions) Complete(flags *pflag.FlagSet) error {
+	var err error
+	o.clientSet, err = kubeutil.GenClientSet(flags)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+// Validate makes sure provided values for ClusterInfoOptions are valid
+func (o *ClusterInfoOptions) Validate() error {
+	return nil
+}
+
+func (o *ClusterInfoOptions) Run() (err error) {
+	// 1. list openyurt cluster node
+	key := projectinfo.GetEdgeWorkerLabelKey()
+	Nodes, err := o.clientSet.CoreV1().Nodes().List(metav1.ListOptions{LabelSelector: key})
```

Comment:
i think we need to get all of nodes in the cluster, not filter by `edge worker label`. so how about delete `LabelSelector: key`

Reply:
Without the filter，it will be show all kubernetes cluster node. If you indeed want this, i will amend it.
The print message will change from "openyurtl cluster nodes list" to "kubernetes cluster nodes list"

Reply:
I think we need list all the nodes of cluster,  so We need to add an slice to show nodes that are not the other two types(cloud node and edge node).

Reply:
We can also keep the print message as `openyurt cluster nodes list` that mean all nodes in cluster

Reply:
So that will four node list , `openyurt cluster nodes list` , `cloud nodes list`, `edge nodes list`, `other nodes list` ?

Reply:
ok